Emby.Server.Implementations/Library/Resolvers/TV/SeasonResolver.cs

```diff
@@ -54,7 +54,8 @@ protected override Season Resolve(ItemResolveArgs args)
                 {
                     IndexNumber = seasonParserResult.SeasonNumber,
                     SeriesId = series.Id,
-                    SeriesName = series.Name
+                    SeriesName = series.Name,
+                    Path = seasonParserResult.IsSeasonFolder ? path : args.Parent.Path
```

Comment:
This line is causing issues for folder structures with season names. E.g. `/Drive/Show/Season 3 - Season Name` will end up with `isSeasonDirectory=false`. In 10.9.3<= this worked perfectly fine, now is uses the series path causing all episodes to merge into the last season.

You can add `[InlineData("/Drive/Show/Season 3 - Season Name", 3, true)]` to the SeasonFolderTests to reproduce the issue.